treeopt/modules/metamodell.py

```diff
@@ -0,0 +1,36 @@
+import smt.surrogate_models as smt
+
+def RBF(xt, yt):
+    """Function, that trains a metamodell based on the radial basis function algorithm
```

Comment:
Could you please use the [Sphinx Docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html), so that this description is automatically compiled into the documentation at https://treeopt.readthedocs.io/en/latest/api.html? This applies to all functions/modules/scripts.

Reply:
Done :)